### PR TITLE
Fix get_full_container_list function

### PR DESCRIPTION
**Pull request by barykine**

### CODE REVIEW

File: objectstore/databasedumps.py [hunk 1]
```diff
@@ -112,7 +112,7 @@ def remove_old_dumps(connection, container: str, days=None):
             objectstore.delete_object(connection, container, o_info)
 
 
-def download_database(connection, container: str, target: str=""):
+def download_database(connection, container: str, target: str = ""):
```
Comment (JKoetsier) [inline objectstore/databasedumps.py:115]: Why the added spaces? PEP8 is without the spaces

File: objectstore/objectstore.py [hunk 2]
```diff
@@ -64,7 +64,7 @@ def make_config_from_env():
     return OBJECTSTORE
 
 
-def get_connection(store_settings: dict={}) -> Connection:
+def get_connection(store_settings: dict = {}) -> Connection:
```
Comment (JKoetsier) [inline objectstore/objectstore.py:67]: Spaces here too

File: objectstore/objectstore.py [hunk 3]
```diff
@@ -99,18 +99,16 @@ def get_connection(store_settings: dict={}) -> Connection:
...
+    _, objects = conn.get_container(container, **kwargs)
+    for object_info in objects:
         yield object_info
 
-    while len(lastpage) == limit:
-        # keep getting pages..
-        kwargs['marker'] = lastpage['name']
-        _, lastpage = conn.get_container(container, **kwargs)
-        for object_info in lastpage:
+    while len(objects) == limit:
```
Comment (JKoetsier) [inline objectstore/objectstore.py:110]: Can we add some comments about the logic? The necessity for the while loop isn't immediately clear at first sight.

File: objectstore/databasedumps.py (hunk 1 above)
Comment (barykine) [reply objectstore/databasedumps.py:115]: This is a part of flake8 cleanup.

File: objectstore/objectstore.py (hunk 2 above)
Comment (barykine) [reply objectstore/objectstore.py:67]: The same

File: objectstore/objectstore.py (hunk 3 above)
Comment (barykine) [reply objectstore/objectstore.py:110]: Done

File: objectstore/databasedumps.py (hunk 1 above)
Comment (barykine) [reply objectstore/databasedumps.py:115]: PEP8: "When combining an argument annotation with a default value, however, do use spaces around the = sign:" https://www.python.org/dev/peps/pep-0008/

Comment (JKoetsier) [reply objectstore/databasedumps.py:115]: Ah I see, my mistake. Didn't realise it was different for annotated default arguments compared to unannotated default arguments. IDE formatting makes dumb ;)

**Review by JKoetsier — APPROVED** [review]

**Pull request merged**